scoap3/articles/models.py

```diff
@@ -27,14 +39,23 @@ class Article(models.Model):
     related_materials = models.ManyToManyField(
         "misc.RelatedMaterial", related_name="related_articles", blank=True
     )
-    _files = models.ManyToManyField(ArticleFile, blank=True)
     _created_at = models.DateTimeField(auto_now_add=True)
     _updated_at = models.DateTimeField(auto_now=True)
 
     class Meta:
         ordering = ["id"]
 
 
+class ArticleFile(models.Model):
+    article_id = models.ForeignKey("articles.Article", on_delete=models.CASCADE)
```

Comment:
pass also `related_name='files' ` to ForeignKey, so it can be access from the article as `article.files`